Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
-ruby '2.2.3'
```

Comment:
Hmmm. Would the buildpacks affect this? I thought the Gem version is specified in our build container.

I will go ahead and merge per your suggestion and we can take the general issue up later.